RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -1540,74 +1553,78 @@ PixelQuintupletsBuffer<alpaka_common::DevHost>& Event::getPixelQuintuplets(bool
   return pixelQuintupletsInCPU_.value();
 }
 
-TrackCandidatesBuffer<alpaka_common::DevHost>& Event::getTrackCandidates(bool sync) {
-  if (!trackCandidatesInCPU_) {
-    // Get nTrackCanHost parameter to initialize host based trackCandidatesInCPU_
+const TrackCandidatesHostCollection& Event::getTrackCandidates(bool sync) {
```

Comment:
> In this case where everything is copied maybe it makes more sense to use `CopyToHost` like in my PR:

This method is not copying everything to simply switch to a full `CopyToHost`.

- Not all columns (even partially): the `lowerModuleIndices`, `center{X,Y}` and `radius` are omitted
    - this is a fractionally small part, I don't have strong arguments not to copy these columns as well
- the copy is only for the created objects, not for the full size preallocated on the device
    - I still have some preference to preserve the ability to copy the needed size, not everything. But perhaps, considering that this almost full copy variant is not really in a performance critical application, this can indeed be switched to a full copy with enough repetition of "why not"